{{cookiecutter.project_slug}}/README.md

```diff
@@ -10,6 +10,14 @@
 
 ## Features
 
+{% if cookiecutter.include_src_folder in ["yes", "y"] -%}
+* src layout: Is a good practice moving the Python code that is
+  intended to be importable of your package in a *src* directory. This
+  package structure is technically called a “regular package” in Python,
```

Comment:
any source about where it is calling src as regular package?

Reply:
Let me look it up, I might also be translating it wrong.